UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -927,7 +1023,24 @@
       };
 
       eventbus.on('asset:modified', function(){
+        readOnly = authorizationPolicy.formEditModeAccess();
+
+        if (busStopTypeSelected == 7 && !readOnly && readOnly !== undefined &&
+              selectedMassTransitStopModel.getCurrentAsset() !== undefined && selectedMassTransitStopModel.getCurrentAsset().id === undefined) {
```

Comment:
Apply "_isUndefined" function here and the new method to validate it is a service point stop

Reply:
@marquesfm Check modification on this code